packages/caliper-vscode-extension/artifacts/json-schema/hyperledger-caliper-ethereum-network-configuration.schema.json.json

```diff
@@ -0,0 +1,129 @@
+{
```

Comment:
For now, let's remove this schema from the PR. Let's focus only on the runtime configuration schema.